src/parser/converts/attr.ts

```diff
@@ -922,6 +923,9 @@ function buildProcessExpressionForExpression(
   typing: string | null,
 ): (expression: ESTree.Expression) => ScriptLetCallback<ESTree.Expression>[] {
```

Comment:
@ota-meshi 

Should I use `TSESTree` here? Write `ESTree.Expression | TSESTree.Expression` everytime is hassle, and since the type information is only needed in a few places, I thought it would be okay to leave the type information as it is. What do you think?

Reply:
I think it's OK to leave it as is for now :+1:. I think that acorn-typescript nodes and TSESTree are not compatible, so just using TSESTree will not be a complete solution.